packages/decentraland-ecs/src/decentraland/Input.ts

```diff
@@ -76,16 +87,29 @@ export class Input {
     return this.internalState
   }
 
-  private subscriptions: Record<InputEventKind, Array<(e: LocalPointerEvent) => void>> = {
-    BUTTON_DOWN: [],
-    BUTTON_UP: []
+  private subscriptions: Record<ActionButton, Record<InputEventKind, Array<Subscription>>> = {
```

Comment:
nit: Initialize with a foreach statement.
```suggestion
  private subscriptions: Record<ActionButton, Record<InputEventKind, Array<Subscription>>> = Object.keys(ActionButton).map(button => ({ 
  [button]: {
      BUTTON_DOWN: [],
      BUTTON_UP: []
    }})).reduce((accumulator, current) => ({ ...accumulator, ...current}))
```